configure.ac

```diff
@@ -267,6 +267,7 @@ AM_CONDITIONAL([MAKE_TESTS], [test x$have_check = xyes])
 
 PKG_CHECK_MODULES([GLIB], [glib-2.0 >= 2.16],[],[AC_MSG_ERROR(GLib 2.16 or newer not found. Install missing package and re-run)])
 PKG_CHECK_MODULES([JSONC], [json-c], [have_jsonc='yes'], [have_jsonc='no'])
+PKG_CHECK_MODULES([ZLIB], [zlib], [AC_DEFINE([HAVE_ZLIB], [1], [Found ZLIB])], [])
```

Comment:
Looks like on Centos the pkg-config macro requires the last `[]` to be filled-in with something so that it doesn't error out. Just set it to `[AC_MSG_NOTICE(No zlib found)]`